src/overlays/actors/ovl_Dm_Char04/z_dm_char04.c

```diff
@@ -28,18 +30,125 @@ const ActorInit Dm_Char04_InitVars = {
     (ActorFunc)DmChar04_Draw,
 };
 
-#endif
+void func_80AABC40(SkelAnime* arg0, AnimationInfo* animation, u16 arg2) {
+    f32 phi_fv1;
+
+    animation += arg2;
+
+    if (animation->frameCount < 0.0f) {
+        phi_fv1 = Animation_GetLastFrame(animation->animation);
+    } else {
+        phi_fv1 = animation->frameCount;
+    }
+    Animation_Change(arg0, animation->animation, animation->playSpeed, animation->startFrame, phi_fv1, animation->mode,
+                     animation->morphFrames);
+}
+
+static AnimationInfo sAnimations[] = {
+    { &gameplay_keep_Anim_02B2E8, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+    { &gameplay_keep_Anim_029140, 1.0f, 0.0f, -1.0f, ANIMMODE_LOOP, 0.0f },
+};
+
+Color_RGBAf sPrimColors[] = { { 250.0f, 255.0f, 230.0f, 255.0f },
+                              { 10.0f, 10.0f, 40.0f, 255.0f },
+                              { 255.0f, 235.0f, 220.0f, 255.0f } };
```

Comment:
This would look better with an extra trailing comma and formatting again
```suggestion
Color_RGBAf sPrimColors[] = { { 250.0f, 255.0f, 230.0f, 255.0f },
                              { 10.0f, 10.0f, 40.0f, 255.0f },
                              { 255.0f, 235.0f, 220.0f, 255.0f }, };
```